packages/scripts/src/application/defaults.ts

```diff
@@ -1,13 +1,8 @@
 import type { IBuildCommandOptions } from './types';
 
-export const DEFAULT_EXTERNAL_FEATURES_PATH = 'external-features.json';
-
 export const buildDefaults = {
     mode: 'production',
```

Comment:
not sure keeping this file now improves the readability, considering most code was removed